composer.json

```diff
@@ -1,43 +1,44 @@
 {
-    "name": "flowpack/media-ui",
-    "description": "This module allows managing media assets including pictures, videos, audio and documents.",
-    "type": "neos-package",
-    "require": {
-        "php": "^8.1",
-        "neos/media": "^7.3 || ~8.0",
-        "neos/neos": "^7.3 || ~8.0",
-        "neos/neos-ui": "^7.3 || ~8.0",
-        "t3n/graphql": "^2.1 || ^3.0.2",
-        "t3n/graphql-upload": "^1.0 || ^2.0"
-    },
-    "require-dev": {
-        "phpunit/phpunit": "^9.5"
-    },
-    "suggest": {
-      "phpstan/phpstan": "For running code quality checks",
-      "flowpack/neos-asset-usage": "Allows filtering unused assets and other related features",
-      "flowpack/entity-usage-databasestorage": "Required for the asset usage features"
-    },
-    "scripts": {
-      "test": "../../../bin/phpunit --enforce-time-limit --bootstrap ../../Libraries/autoload.php --testdox Tests",
-      "test:ci": "phpunit --enforce-time-limit --bootstrap vendor/autoload.php --testdox Tests",
-      "codestyle": "phpstan analyse --autoload-file ../../Libraries/autoload.php",
-      "codestyle:ci": "phpstan analyse"
-    },
-    "license": "GPL-3.0-or-later",
-    "autoload": {
-        "psr-4": {
-            "Flowpack\\Media\\Ui\\": "Classes"
-        }
-    },
-    "autoload-dev": {
-        "psr-4": {
-            "Flowpack\\Media\\Ui\\Tests\\": "Tests"
-        }
-    },
-    "config": {
-        "allow-plugins": {
-            "neos/composer-plugin": true
-        }
+  "name": "flowpack/media-ui",
+  "description": "This module allows managing media assets including pictures, videos, audio and documents.",
+  "type": "neos-package",
+  "require": {
+    "php": ">=8.1",
+    "neos/media": "^8.3",
+    "neos/neos": "^8.3",
+    "neos/neos-ui": "^8.3",
+    "webonyx/graphql-php": "^15",
+    "wwwision/types": "^1.6",
+    "wwwision/types-graphql": "^1.3"
+  },
+  "require-dev": {
+    "phpunit/phpunit": "^9.5"
+  },
+  "suggest": {
+    "phpstan/phpstan": "For running code quality checks",
```

Comment:
i love u

Reply:
edit: no for level 2 php stan it just: i like you

... sorry

Reply:
Will bump it to 3. Level 4 has still to many issues.